packages/core/src/config/config.ts

```diff
@@ -415,6 +417,7 @@ export class Config {
   licenseKey?: string;
   region?: Region;
   'features.openapi': Record<string, any>;
+  'features.mocks'?: Record<string, any>;
```

Comment:
Do we have to put something like 
```
this['features.mocks'] = rawConfig['features.mocks'] || {};
```
inside the constructor?

Reply:
maybe, could you help with this?

Reply:
Resolved